pandas/tests/test_algos.py

```diff
@@ -11,8 +11,9 @@
 
 from pandas._libs import (
     algos as libalgos, groupby as libgroupby, hashtable as ht)
-from pandas.compat import PY2, lrange, range
-from pandas.compat.numpy import np_array_datetime64_compat
+from pandas.compat import lrange, range
```

Comment:
see my comment above

Reply:
Again, if I don't fix the regex here, the tests will fail.